### ISSUE TITLE
Getting constant notice that Bleeding Edge is not running the latest version

### ISSUE THREAD

**Issue by oskosk** [Automattic/jetpack-beta#36]
If I activate the Bleeding Edge branch, I always get a notice about it not running the latest version. 

**Screenshot**

![image](https://user-images.githubusercontent.com/746152/32456124-3a5987c6-c303-11e7-9ed6-0c7e7bc37fb8.png)

**Screencap**

![bleedingedge](https://user-images.githubusercontent.com/746152/32456071-0a24cb9c-c303-11e7-918c-08168760e620.gif)

**Comment by enejb** [comment]
Yeah that definitely looks like a bug. I haven't see that before. I wonder if it has to do something ho w we get the latest version of Bleeding edge.

**Comment by designsimply** [comment]
I managed to run into this problem and then also completely break my setup (white screens, 500 errors) after clicking the "Update" button next to "Bleeding Edge".

<strong>Video:</strong> [2m54s](https://cloudup.com/cJncxSgwtxD)

Steps to reproduce:

1. Start with a site with the Jetpack and Jetpack Beta Tester plugins installed.
1. Go to WP Admin > Jetpack > Jetpack Beta.
1. Find the "Currently Running" section.
1. Verify autoupdates are on.
1. Note the version ("Bleeding Edge | 5.6-beta-16552-576a76c-master" in my case).
1. Note the status under "Bleeding Edge" near the bottom ("last updated 3 hours ago" in my case).
1. Click the "Update" button next to "Bleeding Edge" near the top.

<strong>Result:</strong> first, I expected the version to be 5.8-something if "Bleeding Edge" is active and autoupdates are on but the version is 5.6-something (old). Next, I managed to break everything! Completely blank white screens for front-end and wp-admin after clicking WP Admin > Jetpack > Jetpack Beta > Bleeding Edge > "Update".

<img src="https://user-images.githubusercontent.com/1119271/35696752-88008f70-0745-11e8-9240-a81690bbcf7f.png" />
<sup>Tested on alittletestblog.com with WP 5.0-alpha-42373 and Jetpack 5.6-beta-16552-576a76c-master on WP Engine.</sup>

<strong>Note:</strong> I tried to find logs but the error logs section in wpengine says they're empty ([screenshot](https://user-images.githubusercontent.com/1119271/35696752-88008f70-0745-11e8-9240-a81690bbcf7f.png)).
